src/recap.c

```diff
@@ -85,6 +88,7 @@ void show_app_help(char *app_name) {
   fprintf(stdout, "\t-p filename\t Path to the pcap file name\n");
   fprintf(stdout, "\t-f filename\t Path to the capture db or pipe\n");
   fprintf(stdout, "\t-i interface\t Interface name to save to db\n");
+  fprintf(stdout, "\t-t\t\t Capture from interface using libpcap\n");
```

Comment:
Why `-t` :tea: ?

I'm mainly asking because in PR #348, I used the exact same character (`-t` for `transaction`) (great minds think alike), see https://github.com/nqminds/edgesec/blob/736e0917212ffc27e4a9e5fa25d03d9a22934102/src/recap.c#L122-L132 :)

I did try looking at `getopts_long(` so that I could use `--transaction`, but that's a glibc specific extension, so we may struggle to get it working properly on FreeBSD.

Reply:
It was a random selection. I can change -t to something else.

Reply:
> It was a random selection

I think being vegan may have given you psychic superpowers, so you read my mind and realised predicted which option char I was using :wink: :brain: